docs/content/en/docs/getting-started/_index.md

```diff
@@ -27,16 +27,13 @@ Below, we outline various options to kickstart SOARCA. The latest pre-compiled r
 make build && ./build/soarca
 {{< /tab >}}
 {{< tab header="Linux" lang="sh" >}}
-wget https://github.com/COSSAS/SOARCA/releases/download/SOARCA_1.0.0/SOARCA_1.0.0_linux_amd64.tar.gz  && tar -xvf SOARCA* && ./SOARCA
+wget <https://github.com/COSSAS/SOARCA/releases/download/SOARCA_1.0.0/SOARCA_1.0.0_linux_amd64.tar.gz>  && tar -xvf SOARCA* && ./SOARCA
```

Comment:
This is not correct 
<img width="970" alt="image" src="https://github.com/user-attachments/assets/5c8aeab2-6d3d-4768-ac58-7bf6822a2567">